src/MUST_analysis_tools/must_cp_analysis_readme.md

```diff
@@ -0,0 +1,32 @@
+## Introduction
+
+This code is data analysis for the UDP output from the MUST sensor in the CDA research task 4 Freight cooperative perception project. Testing for that project was completed on September 12th. Testing involved one instrumented vehicle, a black rental car with a novatel GPS unit installed inside and an antenna mounted on the roof. UDP and GPS ground truth data are available on google drive (links below). Raw data (pure video and timestamp files) are available on google drive, but require processing through the model code. The raw data allows you to 
+
+The 'short' testing data and associated python file `MUST_CP_short_analysis.py` contains the instrumented ground truth vehicle, and is intended for testing the position, velocity, and heading of that specific vehicle. The 'long' testing data and associated python file `MUST_CP_long_analysis.py` does not contain the instrumented vehicle, and is intended for analyzing the class/ID switching behavior and manually verifying bounding boxes. 
+
+The other files, `MUST_CP_calibration_analysis.py` and `compute_homography_image_warp.py` are for calibrating the camera without a checkerboard. For the original testing, we did not have any intrinsics computed nor a way to being a checkerboard up to the camera to do it properly. `compute_homography_image_warp.py` contains point correspondences and code to compute the intrinsics. The point correspondences were manually defined and entered, but the opencv function to compute the intrinsics was not working with so few points. A simulated annealing algorithm was created and tuned to find intrinsics through step-by-step optimization, and it works pretty well. The output intrinsics and homography transformation can be fed into `MUST_CP_calibration_analysis.py` to validate the effectiveness of that calibration vs. the original calibration from UW. This is basically the same as `MUST_CP_short_analysis.py`, but set up to read in extended data with image coordinates for the vehicles and process them for testing the calibration. 
+
+## Setup
+
+1. Installation folder
```

Comment:
The markdown result does not have any structure to be clear:
![image](https://github.com/user-attachments/assets/f638e4ae-30ac-4f03-bd82-720794bfc83a)